### PR TITLE
ul blog improvements

### PR DESCRIPTION
**Pull request by isemona**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL approved by Dev Advocacy
- [ ] The content has been run through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist
- [ ] Images are compressed appropriately
- [ ] Social image previews well on Twitter and LinkedIn
- [ ] Tech review request (developer advocate or domain expert)
- [ ] Review request for editorial/grammar/clarity (developer advocate or Colton)

### CODE REVIEW

**Review by edunham — APPROVED** [review]
👍 from a "doesn't break anything I have insight into" perspective. This is not a technical review of the new code you've added.

File: _source/_posts/2024-04-30-express-universal-logout.md [hunk 1]
```diff
@@ -183,22 +185,72 @@ universalLogoutRoute.post('/global-token-revocation', async (req, res) => {
...
+
+  // Find the user
+  const newRequest:IRequestSchema = req.body;
+  const { email } = newRequest.sub_id;
+  const user = await prisma.user.findFirst({
+    where: {
+      email: email
+    },
+  });
+  return res.sendStatus(httpStatus);
```
Comment (alisaduncan) [inline _source/_posts/2024-04-30-express-universal-logout.md]: ?? am i missing a paren indention? This looks like you are returning out of the method first, then there's code that will never get hit to check if the user doesn't exist. I don't understand. is the intent for the 404 check to move higher?

File: _source/_posts/2024-04-30-express-universal-logout.md [hunk 2]
```diff
@@ -183,22 +185,72 @@ universalLogoutRoute.post('/global-token-revocation', async (req, res) => {
...
+
+  // 404 User not found
+  if (!user) {
+    res.sendStatus(404);
+  }
+
+});
+
+universalLogoutRoute.use((err,req,res,next) => {
+  if(err){
```
Comment (alisaduncan) [inline _source/_posts/2024-04-30-express-universal-logout.md:251]: What's an error and why are they all 404?

File: _source/_posts/2024-04-30-express-universal-logout.md [hunk 3]
```diff
@@ -602,7 +668,7 @@ if (!res.ok)
 }}     
 ```
 
-The onNewTask function will now look like this:
+The onNewTask function will now look like this with a change made only to the `onNewTAsk` function:
```
Comment (alisaduncan) [inline _source/_posts/2024-04-30-express-universal-logout.md]: You should re-evaluate this sentence. It seems repetitive. Also change casing for `onNewTAsk`

File: _source/_posts/2024-04-30-express-universal-logout.md [hunk 4]
```diff
@@ -635,8 +701,8 @@ export const Todos = () => {
         });
 
         if (!res.ok){if (res.status === 401) {
-// Redirect user back to the sign in page
-window.location.href = '/';
+        // Redirect user back to the sign in page
+        window.location.href = '/';
```
Comment (alisaduncan) [inline _source/_posts/2024-04-30-express-universal-logout.md:709]: out of curiosity, why are using window API here to redirect to home. I thought this is handled already inside the todo app, can't we use React Router?

**Review by alisaduncan — COMMENTED** [review]
changes requested

File: _source/_posts/2024-04-30-express-universal-logout.md (hunk 1 above)
Comment (isemona) [reply _source/_posts/2024-04-30-express-universal-logout.md]: fixed this thanks for catching!

File: _source/_posts/2024-04-30-express-universal-logout.md (hunk 2 above)
Comment (isemona) [reply _source/_posts/2024-04-30-express-universal-logout.md:251]: wow I'm definitely missing here too

`return res.sendStatus(httpStatus);
});`

File: _source/_posts/2024-04-30-express-universal-logout.md (hunk 4 above)
Comment (isemona) [reply _source/_posts/2024-04-30-express-universal-logout.md:709]: will pick up on this in next PR

**Pull request merged**